backend/src/main/java/ca/bc/gov/hlth/mohums/userSearch/user/UserSpecifications.java

```diff
@@ -41,21 +42,50 @@ public Specification<UserEntity> emailLike(String email) {
                 criteriaBuilder.like(criteriaBuilder.lower(root.get("email")), formatQueryParam(email));
     }
 
-    /*
-    This method doesn't use formatQueryParam method because according to Keycloak API specification,
-     the default search behaviour (when using search query param) is prefix-based
+    /**
+   To keep the basic search functionality intact, most of the logic is copied from this class:
+   https://github.com/keycloak/keycloak/blob/main/model/jpa/src/main/java/org/keycloak/models/jpa/JpaUserProvider.java
```

Comment:
Maybe update this link to point to the specific version at this point in time, so https://github.com/keycloak/keycloak/blob/24.0.4/model/jpa/src/main/java/org/keycloak/models/jpa/JpaUserProvider.java